app/views/items/browse_facet.html.erb

```diff
@@ -0,0 +1,40 @@
+<h2>Browse by <%= @browse_facet_info["label"] %></h2>
+
+<nav>
+  <ul class="nav nav-tabs bottom-buffer" role="tablist">
+    <% Facets.facet_info.each_with_index do |(facet_name, facet_info), index| %>
+      <li role="presentation" <%= "class=active" if facet_name == @browse_facet %>>
+        <% if facet_name == @browse_facet %>
+          <a href="#"><%= facet_info["label"] %></a>
+        <% else %>
+          <% id = datify_name facet_name %>
```

Comment:
btw, if you have any better ideas of how to do this, I would be so interested.  Once you reminded me how the hyphenated names are working, all the pain of trying to get the facets to work came flooding back to me, haha.